http_check/check.py

```diff
@@ -267,13 +267,14 @@ def send_status_down(loginfo, message):
                            % (str(e), length))
             raise
 
+        tags_list = list(tags)
+        tags_list.append('url:%s' % addr)
```

Comment:
Yea, it's used later, outside the if. I got an error about tag_list not defined, and that change fixed it